docs/developers-guide.md

```diff
@@ -25,16 +25,16 @@ If you have problems with your development environment, make sure that you are n
 These are the set of tools which are required in order to complete any build of the Metabase code. Follow the links to download and install them on your own before continuing.
 
 1. [Clojure (ttps://clojure.org)](https://clojure.org/guides/getting_started) - install the latest release by following the guide depending on your OS
-2. [Java Development Kit JDK (https://adoptopenjdk.net/releases.html)](https://adoptopenjdk.net/releases.html) - you need to install JDK 8 (to build) and JDK 11 (to run Metabase) - more about [Java versions](./operations-guide/java-versions.md)
+2. [Java Development Kit JDK (https://adoptopenjdk.net/releases.html)](https://adoptopenjdk.net/releases.html) - you need to install JDK 8 (to build) and JRE 11 (to run Metabase) - more about [Java versions](./operations-guide/java-versions.md)
```

Comment:
Can we completely remove JRE 11 here? JDK 8 should be sufficient to run the .jar.

Reply:
you're absolutely right, with just JDK 8 works for both compiling and running